src/pages/About/AboutPage.style.js

```diff
@@ -0,0 +1,399 @@
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      width: 20.625rem;
+    }
+  `,
+  LogoWrapper: styled(Logo)`
+    width: 28.12488rem;
+    height: 7.97rem;
+    @media (max-width: 1279px) {
+      width: 20.625rem;
+
+      margin-bottom: 5rem;
+    }
+  `,
+  AboutPRContainer: styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 4rem;
+    @media (max-width: 1279px) {
+      gap: 2rem;
+    }
+  `,
+  RedTextWrapper: styled.span`
+    color: var(--red);
+  `,
+  NameWrapper: styled.div`
+    ${bold24}
+    color: var(--grey100);
+    @media (max-width: 1279px) {
+      ${m_bold24}
+    }
+  `,
+  HistoryWrapper: styled.div`
+    width: 34.125rem;
+    margin-bottom: 17.5rem;
+    ${reg18};
+    color: var(--grey100);
+    @media (max-width: 1279px) {
+      width: 20.625rem;
+      margin-bottom: 0rem;
+      ${m_reg18}
+    }
+  `,
+  SubTitleContainer: styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 3rem;
+    @media (max-width: 1279px) {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      width: 100%;
+      gap: 2rem;
+    }
+
+    align-items: start;
+  `,
+
+  TextWrapper: styled.div`
+    width: 16.1875rem;
+    text-align: start;
+    white-space: pre-wrap;
+    ${reg18};
+    color: var(--grey100);
+    @media (max-width: 1279px) {
+      width: 20.9375rem;
+      margin-top: 2rem;
+      ${m_reg18}
+    }
+  `,
+
+  SubTitleContentContainer: styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+    gap: 0.5rem;
+  `,
+  SubTitleWrapper: styled.div`
+    ${bold36}
+    color: var(--grey100);
+    @media (max-width: 1279px) {
+      width: 20.625rem;
+      margin-top: 10rem;
+      ${m_bold36}
+    }
+  `,
+  SubTitleKrWrapper: styled.div`
+    ${reg18}
+    color: var(--grey300);
+    @media (max-width: 1279px) {
+      ${m_reg18}
+    }
+  `,
+  CardContainer: styled.div`
+    display: flex;
+    gap: 1.75rem;
+    @media (max-width: 1279px) {
+      display: flex;
+      flex-direction: column;
+    }
+  `,
+
+  HiddenCardContainer: styled.div`
+    display: flex;
+    gap: 1.75rem;
+    @media (max-width: 1279px) {
+      display: none;
+    }
+  `,
+
+  CardContentContainer: styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 10.5rem;
+    height: 10.5rem;
+    border-radius: 1rem;
+    background-color: var(--grey500);
+    box-sizing: border-box;
+    box-shadow: 0px 0px 20px 0px rgba(0, 0, 0, 0.4);
+    @media (max-width: 1279px) {
+      width: 20.9rem;
+      padding: 2.5rem 3.75rem;
+    }
+  `,
+  CardNameWrapper: styled.div`
+    ${bold24}
+    color:var(--white);
+  `,
+  DepartmentContainer: styled.div`
+    display: flex;
+    flex-direction: column;
+  `,
+  DepartmentContentContainer: styled.div`
+    display: flex;
+    flex-direction: column;
+    width: 46.0625rem;
```

Comment:
```suggestion
    max-width: 46.0625rem;
    width: 40.0625rem;
```
패딩 고려해서 width 줘야 해요!